theme/open/views/imprint/imprint.hbs

```diff
@@ -1,10 +1,10 @@
 <div class="section-imprint {{#unless loggedin}}loggedout{{/unless}}">
 	{{#unless loggedin}}
-	<h4>Impressum</h4>
+	<h1 class="h4">Impressum</h1>
 	{{/unless}}
-	<h5>
+	<h2 class="h5">
 		<strong>HPI Schul-Cloud - Hasso-Plattner-Institut für Digital Engineering gGmbH</strong>
-	</h5>
+	</h2>
 	<p>Prof.-Dr.-Helmert-Str. 2-3
```

Comment:
It used to be the case that an <address> element was only supposed to be used to represent the contact information of the document's author. In the latest spec versions however, its definition has been updated so it can now be used to mark up arbitrary addresses.
The text in the <address> element usually renders in italic, and browsers will always add a line break before and after the <address> element.

Reply:
I am not sure if it would be appropriate in our case (Imprint), possibly in the footer where `mailto `is used

Reply:
I was trying out with the address element, and it seems fine, not changing anything regarding line breaks. so we could do it for the sake of semantics I'd say - but no must
<img width="617" alt="Screen Shot 2020-07-16 at 15 20 20" src="https://user-images.githubusercontent.com/7567881/87675644-eb3e8480-c777-11ea-8835-0c659422ed12.png">